sql/core/src/main/scala/org/apache/spark/sql/execution/joins/JoinCodegenSupport.scala

```diff
@@ -95,4 +95,27 @@ trait JoinCodegenSupport extends CodegenSupport with BaseJoinExec {
       }
     }
   }
+
+  /**
+   * Splits variables based on whether it's used by condition or not, returns the code to create
```

Comment:
I assume this just moves the code without any change.

Reply:
Yes, move `splitVarsByCondition` method into JoinCodegenSupport, so it can be used in ShuffledHashJoinExec